service/src/test/java/uk/gov/hmcts/reform/fpl/utils/ObjectHelperTest.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.ccd.model.Organisation;
+import uk.gov.hmcts.reform.ccd.model.OrganisationPolicy;
+import uk.gov.hmcts.reform.fpl.enums.C2ApplicationType;
+import uk.gov.hmcts.reform.fpl.enums.State;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.ManagedDocument;
+import uk.gov.hmcts.reform.fpl.model.SentDocuments;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+public class ObjectHelperTest {
+
+    @Test
+    public void testEnum() throws Exception {
+        CaseData caseData = CaseData.builder()
+            .state(State.CLOSED)
+            .build();
+
+        assertEquals(State.CLOSED, ObjectHelper.getFieldValue(caseData, "state", State.class));
+    }
+
+    @Test
+    public void testList() throws Exception {
+        List<Element<SentDocuments>> testList = List.of(
+            element(SentDocuments.builder().partyName("test1").build()),
+            element(SentDocuments.builder().partyName("test2").build()),
+            element(SentDocuments.builder().partyName("test3").build()));
+
+        CaseData caseData = CaseData.builder().documentsSentToParties(testList).build();
+
+        assertEquals(testList, ObjectHelper.getFieldValue(caseData, "documentsSentToParties", List.class));
+    }
+
+    @Test
+    public void testNested() throws Exception {
```

Comment:
add a test for returning a List from a nested property?

Reply:
added `testListInNestedField` test